modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationTemplateLocalServiceImpl.java

```diff
@@ -289,8 +291,10 @@ else if (Validator.isNull(fromName)) {
 				siteDefaultLocale, null, notificationType, object);
 		}
 
-		String to = _formatContent(
-			notificationTemplate.getTo(user.getLocale()), user.getLocale(),
+		String to = _formatTo(notificationTemplate.getTo(user.getLocale()));
+
+		to = _formatContent(
+			to, user.getLocale(),
 			NotificationTermContributorConstants.RECIPIENT, notificationType,
 			object);
```

Comment:
Can you move the call to _formatContent inside _formatTo? The result would be something like:
```
private String _formatTo(String to, ...) {
    ... // current code.

    return _formatContent(StringUtil.merge(emails, ","), ...);
}
```
then, the calling code would be much simpler:
```
String to = _formatTo(notificationTemplate.getTo(user.getLocale()), user.getLocale(), ...);
```

Reply:
@adolfopa, I updated the code. Thanks!